### PR TITLE
Fix broken links

### PR DESCRIPTION
**Pull request by fatsheep9146**
Fix broken links

Signed-off-by: Ziqi Zhao <zhaoziqi@qiniu.com>

### CODE REVIEW

File: contributors/design-proposals/node/pod-resource-management.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ to Kubernetes. It outlines the implementation and associated rollout plan.
 ## Introduction
 
 Kubernetes supports container level isolation by allowing users
-to specify [compute resource requirements](resources.md) via requests and
+to specify [compute resource requirements](../scheduling/resources.md) via requests and
```
Comment (stp-ip) [inline contributors/design-proposals/node/pod-resource-management.md]: This breaks when moving the file. Why not use: `/contributors/design-proposals/scheduling/resources.md`?

Comment (fatsheep9146) [reply contributors/design-proposals/node/pod-resource-management.md]: You are right. I will change it as you said.

Comment (stp-ip) [reply contributors/design-proposals/node/pod-resource-management.md]: Great thanks.
Also curious sidenote:
Same repo links (as we have here) can just use the relative path.
Cross repo links can be done using something like: "git.k8s.io/community/contributors/design-proposals/scheduling/resources.md"
And issue links can be done using something like: "http://issue.k8s.io/1"

**Comment by stp-ip** [comment]
Now only thing left is do rebase and squash both commits into one. Then we can merge \o/

**Comment by fatsheep9146** [comment]
OK !

**Comment by stp-ip** [comment]
Awesome. Thanks.

**Pull request merged**